src/transformers/pipelines/automatic_speech_recognition.py

```diff
@@ -347,7 +362,30 @@ def postprocess(self, model_outputs, decoder_kwargs: Optional[Dict] = None):
             skip_special_tokens = self.type != "ctc"
             tokens = np.concatenate([outputs["tokens"].numpy() for outputs in model_outputs], axis=-1)
             tokens = tokens.squeeze(0)
-            text = self.tokenizer.decode(tokens, skip_special_tokens=skip_special_tokens)
+
+            if return_timestamps:
+                decoded = self.tokenizer.decode(
+                    tokens, skip_special_tokens=skip_special_tokens, output_char_offsets=True
```

Comment:
The big debate again :sweat_smile: 

Do you think char time stamps are better than word time stamps here or could we maybe just allow both?

Reply:
The more I think about this, the more it seems in the pipelines that maybe instead of being a boolean parameter we could have an enum.

`CHAR` or `WORD`, and if `True` would be understood as `CHAR`.

Reply:
And because for CTC we might not be able to do char we could have other options in the enum ?